cfme/tests/containers/test_containers_smartstate_analysis.py

```diff
@@ -68,9 +93,9 @@ def test_containers_smartstate_analysis(provider, test_item, soft_assert):
                     'Traceback: \n{}'.format(format_exc()))
 
     evm_tail.fix_before_start()
-
     image_obj.perform_smartstate_analysis(wait_for_finish=True)
```

Comment:
This task errored out, causing an exception in the 58z PRT results. please address here if possible.